### PR TITLE
New attempt

### PR DESCRIPTION
**Pull request by kcyt**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -3,4 +3,6 @@
 
 print("Hey World, it's me!")
 
-print("Yo yo, goodbye!")
\ No newline at end of file
+print("Yo yo, Peter goodbye!")
```
Comment (lockyloki) [inline main.py]: What is this?

Comment (kcyt) [reply main.py]: Just an addition

Comment (lockyloki) [reply main.py]: Ok

File: main.py [hunk 2]
```diff
@@ -3,4 +3,6 @@
 
 print("Hey World, it's me!")
```
Comment (lockyloki) [inline main.py:5]: What is ths?

File: main.py [hunk 3]
```diff
@@ -3,4 +3,6 @@
...
 
-print("Yo yo, goodbye!")
\ No newline at end of file
+print("Yo yo, Peter goodbye!")
+
+print("how about this?")
```
Comment (lockyloki) [inline main.py]: Why

File: main.py [hunk 4]
```diff
@@ -3,4 +3,6 @@
 
 print("Hey World, it's me!")
```
Comment (lockyloki) [inline main.py:4]: No

Comment (lockyloki) [inline main.py:4]: Not great

**Review by lockyloki — CHANGES_REQUESTED** [review]
Not great

**Review by lockyloki — APPROVED** [review]
Good

**Pull request merged**